pkg/login/social/generic_oauth.go

```diff
@@ -241,13 +248,15 @@ func (s *SocialGenericOAuth) UserInfo(client *http.Client, token *oauth2.Token)
 
 	role := s.extractRole(&data, rawUserInfoResponse.Body)
 
+	s.log.Debug("Getting role", role, rawUserInfoResponse.Body)
```

Comment:
What is in `rawUserInfoResponse.Body` that is getting logged?

Reply:
Removed.